manager/dispatcher/dispatcher.go

```diff
@@ -572,20 +572,37 @@ func (d *Dispatcher) Tasks(r *api.TasksRequest, stream api.Dispatcher_TasksServe
 			return err
 		}
 
-		select {
-		case event := <-nodeTasks:
-			switch v := event.(type) {
-			case state.EventCreateTask:
-				tasksMap[v.Task.ID] = v.Task
-			case state.EventUpdateTask:
-				tasksMap[v.Task.ID] = v.Task
-			case state.EventDeleteTask:
-				delete(tasksMap, v.Task.ID)
+		// bursty events should be processed in batches and sent out snapshot
+		const eventBatchLimit = 200
```

Comment:
@dongluochen I'm not sure that is how the math works (check your units). This will send a batch every 50ms, with a maximum to 200. If a larger number of tasks is sent, this will continue to send them. There is nothing limiting the rate (which we don't really want, anyways).

Batching really isn't the right way to think of this, since we don't send tasks in batches. We add them in batches to an assignment set.